### PR TITLE
Logo And Color Scheme changes

### PR DESCRIPTION
**Pull request by christinem**
This pull request adds the logo to Courseography and (with the great help of @cchens) changes the color scheme and look of text of the top nav bar and sidebar. New sidebar icon is also added.

### CODE REVIEW

File: hs/Css/Graph.hs [hunk 1]
```diff
@@ -231,14 +231,14 @@ sidebarCSS = do
         width (px 40)
         height100
         float floatLeft
-        backgroundColor purple2
+        backgroundColor $ purple10
```
Comment (cchens) [inline hs/Css/Graph.hs]: Why does this one need the $ operator?

Comment (christinem) [inline hs/Css/Graph.hs]: It doesn't, just fixed it. Thanks for catching it!

File: hs/Css/Graph.hs [hunk 2]
```diff
@@ -330,6 +330,12 @@ sidebarCSS = do
         marginBottom (px 20)
         ":hover" & do
             backgroundColor grey2
+    ".flip" & do
+        "-webkit-transform" -: "scaleX(-1)"
```
Comment (david-yz-liu) [inline hs/Css/Graph.hs]: Clay actually can generate vendor prefixes in many cases, including [transform](https://hackage.haskell.org/package/clay-0.10.1/docs/Clay-Transform.html). Try something like `transform $ scaleX (-1)`

**Comment by david-yz-liu** [comment]
Thanks, @christinem and @cchens! I suggested one change in the CSS. Also, I wonder if perhaps we should use the nav bar colour for the sidebars on the `Grid` page - what do you think?

**Comment by christinem** [comment]
@david-yz-liu I fixed the things that you mentioned. I want to also change the colors on the Check My POSt page because after the background was changed to white, it looks kind of funny now - but if you want to merge this beforehand I'll make a new PR for it.

File: hs/Css/Graph.hs (hunk 2 above)
Comment (cchens) [inline hs/Css/Graph.hs]: @christinem I think the `transform $ scaleX (-1)` was supposed to replace this entire block

Comment (david-yz-liu) [inline hs/Css/Graph.hs]: Yes, that's correct - none of the others need to be there.

**Comment by david-yz-liu** [comment]
@christinem There is one more change to make for this one. You can change the other colours in a separate pull request.

**Comment by christinem** [comment]
@david-yz-liu Fixed! And alright, I'll make it in another PR.

**Comment by david-yz-liu** [comment]
Thanks, @christinem!

**Pull request merged**